drivers/mrf24j40/mrf24j40_getset.c

```diff
@@ -74,38 +74,38 @@ static const uint8_t dbm_to_tx_pow[] = { 0x00, 0x10, 0x18, 0x20, 0x28, 0x30, 0x3
                                          0xc0, 0xd0, 0xd8, 0xe0, 0xe8, 0xf0, 0xf8 };
 
 /* take a look onto datasheet table 3-8 */
-static const int8_t dBm_value[] = {  95, 89, 88, 88, 87, 87, 87, 87, \
-                                     86, 86, 86, 86, 85, 85, 85, 85, \
-                                     84, 84, 84, 84, 84, 84, 83, 83, \
-                                     83, 83, 82, 82, 82, 82, 81, 81, \
-                                     81, 81, 81, 80, 80, 80, 80, 80, \
-                                     80, 79, 79, 79, 79, 79, 78, 78, \
-                                     78, 78, 78, 77, 77, 77, 77, 77, \
-                                     76, 76, 76, 76, 76, 75, 75, 75, \
-                                     75, 75, 75, 74, 74, 74, 74, 73, \
-                                     73, 73, 73, 73, 72, 72, 72, 72, \
-                                     72, 71, 71, 71, 71, 71, 70, 70, \
-                                     70, 70, 70, 70, 70, 69, 69, 69, \
-                                     69, 69, 68, 68, 68, 68, 68, 68, \
-                                     68, 67, 67, 67, 67, 66, 66, 66, \
-                                     66, 66, 66, 65, 65, 65, 65, 65, \
-                                     64, 64, 64, 64, 63, 63, 63, 63, \
-                                     62, 62, 62, 62, 61, 61, 61, 61, \
-                                     60, 60, 60, 60, 60, 59, 59, 59, \
-                                     59, 59, 58, 58, 58, 58, 58, 57, \
-                                     57, 57, 57, 57, 57, 56, 56, 56, \
-                                     56, 56, 56, 56, 55, 55, 55, 55, \
-                                     54, 54, 54, 54, 54, 54, 53, 53, \
-                                     53, 53, 53, 53, 53, 52, 52, 52, \
-                                     52, 52, 52, 51, 51, 51, 51, 51, \
-                                     50, 50, 50, 50, 50, 49, 49, 49, \
-                                     49, 49, 48, 48, 48, 48, 47, 47, \
-                                     47, 47, 47, 46, 46, 46, 46, 45, \
-                                     45, 45, 45, 44, 44, 44, 44, 44, \
-                                     43, 43, 43, 42, 42, 42, 42, 41, \
-                                     41, 41, 41, 41, 41, 40, 40, 40, \
-                                     40, 40, 39, 39, 39, 39, 39, 38, \
-                                     38, 38, 38, 37, 37, 37, 36, 30 };
+static const int16_t dBm_value[] = { -90, -89, -88, -88, -87, -87, -87, -87, \
```

Comment:
looking at the values I don't see why we should double the amount of memory required here

Reply:
plus: why change (or correct) first and last value by 5?

Reply:
even when considering the increase value range for the info struct, its IMHO no necessary to do so here, specifically as the values fit into `int8_t` and `int16_t` would double the required memory <- bad.

Reply:
> looking at the values I don't see why we should double the amount of memory required here

Ack, will fix.

> plus: why change (or correct) first and last value by 5?

That was a small correction from the datasheet values, from the worst (-100dBm) truncated value to the first truncated (-90dBm) value. I'll can put it in a separate PR if you want.

Reply:
no need to put it in a separate PR, was just curious.

Reply:
The whole range from -100dBm to -90dBm is returned as a value of 0, the same with -35dBm to -25dBm. To me it looked weird to have a discontinuity in the values by using the worst/optimal case.

Reply:
yeah, just checked the data sheet, I guess the intention here was to use the mean of those same value upper and lower bounds.

Reply:
sadly there is no simple function to optimise this lookup table a bit or at best completely away.